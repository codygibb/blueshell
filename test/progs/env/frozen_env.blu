func f() {
	return y + 1
}
y := 5
print y
print f()
