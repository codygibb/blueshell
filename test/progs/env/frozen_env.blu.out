5
error: Var_not_found y
